fix_all.js

```
@@ -0,0 +1,13 @@
+const fs = require('fs')
```

ESLint

[error] Insert `;`

(prettier/prettier)